src/modules/tree.ts

```diff
@@ -285,6 +285,30 @@ export default {
     ],
     requiredArgs: 'account',
   },
+  e2e: {
+    description: "Run your VTEX app's integration tests",
+    handler: './apps/e2e',
+    options: [
+      {
+        description: 'Specify test request id',
+        long: 'report',
+        short: 'r',
+        type: 'string',
+      },
+      {
+        description: "Test workspace's apps",
+        long: 'workspace',
+        short: 'w',
+        type: 'boolean',
+      },
+      {
+        description: "[Dangerous] Exposes toolbelt token via 'authToken' environment variable",
```

Comment:
Why one would want to give a token via environment variable in this case? I think this option is unnecessary - and we shouldn't trust the user to do something labeled as `Dangerous`

Reply:
The token option passes on the user's token from the toolbelt to the tester as a TestOptions, which in turn passes on to Cypress via an Cypress_authToken environment variable, which is only available in the Cypress.env test code ('authToken ').
 
I believe it's necessary so Cypress has access to vtex apps, @rafaelbac uses something similar I can ask him why is needed. And for now e2e testing should only be available for internal usage

Reply:
Ohh, I understood now, could you rework the description? I didn't understood this from the current description

Reply:
How about:
```
[Not recommended] Send your personal authorization token to your test session so it's available while running the tests. It can be dangerous because ....
```